src/pdfkit.cs

```diff
@@ -1859,10 +1847,6 @@ interface PdfView :
 		[Export ("highlightedSelections")]
 		PdfSelection [] HighlightedSelections { get; set; }
 
-		[Deprecated (PlatformName.MacOSX, 10, 12)]
```

Comment:
Do these only trigger when submitting iOS apps, or when submitting macOS apps as well? They all seem to be macOS only API.

Reply:
This is an error in our bindings, good catch. There is a NoiOS missing.

Reply:
The same goes for all the other changes in this file, they all look like macOS-only API.